floris/flow_visualization.py

```diff
@@ -215,7 +215,7 @@ def visualize_heterogeneous_cut_plane(
     Args:
         cut_plane (:py:class:`~.tools.cut_plane.CutPlane`): 2D
             plane through wind plant.
-        fi (:py:class:`~.tools.floris_interface.FlorisInterface`): FlorisInterface object.
```

Comment:
@paulf81 just a heads up that this syntax, the `:py:class:` part, is good to keep since it links to the target class in the API docs. See [here](https://nrel.github.io/floris/_autosummary/floris.tools.visualization.visualize_heterogeneous_cut_plane.html#floris.tools.visualization.visualize_heterogeneous_cut_plane) for this particular example.

Reply:
That is good to know, it's a bit random right now the style of of the docstrings depending on by who and when they were written.  What if we made a separate issue/pull to go through and bring all the docstrings to a particular format?

Reply:
No problem, I agree that the docstrings are very inconsistent. I have no issue with leaving them how they are, and I agree that updating the docstrings is outside the scope of this pull request.